zScanner/Services/Tracking/Event.swift

```diff
@@ -12,8 +12,6 @@ enum Event {
     case login
     case logout
     
-    case documentModeSelected(DocumentMode)
```

Comment:
don't remove the tracking event, it can stay there unused :) 
More like add department elected event with Department id/code